.circleci/config.yml

```diff
@@ -229,3 +237,85 @@ jobs:
             else
               printf ":: no errors are found in release\n\n"
             fi
+
+  build_centos7:
+    docker:
+      - image: offical2600hz/kazoo-packager-19:1.0-centos-7
```

Comment:
should be 21 ?

Reply:
oh good point, copy error, also removing the job from the workflow. Will see if that fixes, going to close and reopen this pr to test.